.golangci.toml

```diff
@@ -40,6 +40,8 @@
     "funlen",
     "bodyclose", # Too many false positives
     "godox",
+    "unparam", # Often leads to bad practice
```

Comment:
> What case did you had in mind when claiming it leads to bad practice?
We have some functions in the tests that accept a name and a namespace. It appears that the namespace was always the same. This linter forced us to remove the namespace parameter because it was always the same. 

Also, if you intend to write a function which accepts a param which is not yet used but will be later, you can't. 

Anyway, we've reverted the change.

Reply:
It was complaining even though you passed the namespace as a `_`? That shouldn't happen 🤔